oval_graph/client.py

```diff
@@ -31,6 +31,18 @@ def __init__(self, args):
             self.source_filename)
         self.parts = self.get_src('parts')
 
+    from ._client_create_html_graph import save_html_and_open_html
+    from ._client_create_html_graph import _format_rules_output
+    from ._client_create_html_graph import print_output_message_and_open_web_browser
+    from ._client_create_html_graph import open_web_browser
+    from ._client_create_html_graph import get_src
+    from ._client_create_html_graph import get_save_src
+    from ._client_create_html_graph import _prepare_data
+    from ._client_create_html_graph import prepare_data
+    from ._client_create_html_graph import _merge_report_parts
+    from ._client_create_html_graph import save_html_report
+    from ._client_create_html_graph import _get_part
```

Comment:
It looks like too many imports :D What about importing the whole module instead?